janitor.conf.example

```diff
@@ -17,7 +17,7 @@ redis_location: 'redis://localhost'
 
 # Set the default committer; if unset defaults to current users' preferences
 # (i.e. ~/.gitconfig)
-committer: 'Janitor Bot <janitor@jelmer.uk>'
+committer: 'Janitor <janitor@jelmer.uk>'
```

Comment:
The word "Bot" is intentional here, so that users know that commits by the janitor don't come from a human.

Reply:
Reverted